cmd/dockerregistry/main.go

```diff
@@ -16,6 +18,18 @@ import (
 )
 
 func main() {
+	// Start the profiling server, if requested.
+	profilingAddr := os.Getenv("REGISTRY_PROFILING_ADDR")
```

Comment:
It should bind to loopback by default and the masters have an
additional env var they respect for the addr.